### DISCUSSION TITLE
Where do i change the following? minshare requirments

### DISCUSSION THREAD

**Discussion by BluzBug** [airdcpp/airdcpp-windows#142] [Q&A] [answered]
=== TRAFFIC MANAGER =====================================

     Hello BluzBug,
     your sharesize does not meet the minshare requirements:

     Downloads, Uploads and Searches are blocked.

Any help would be apprciated ;)

**Comment by BluzBug** [comment]
I know it's in Settings .... but beyond that?

**Comment by infectormp** [comment]
This is not related to airdc. You need to share more stuff.

**Comment by maksis** [comment, marked as answer]
It's the `Sharing` tab
